go.mod

```diff
@@ -5,7 +5,7 @@ go 1.13
 require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/giantswarm/apiextensions v0.3.0
-	github.com/giantswarm/certs v0.2.0
+	github.com/giantswarm/certs v0.2.1-0.20200427183231-7d88d71baa73
```

Comment:
Still need to tag the new major version once we sorted it all out. See https://github.com/giantswarm/certs/pull/68.